Components.Tests/RenderTests/RenderTests.cs

```diff
@@ -110,6 +110,23 @@ public void RenderSessionTimeoutControl(Model sut)
             result.ToString().Should().ContainAll("class='wb-sessto'", "inactivity", "reactionTime", "sessionalive", "logouturl", "refreshCallbackUrl", "refreshOnClick", "refreshLimit", "method", "additionalData");
```

Comment:
I'm just noticing because I did the Java PR before :)  but... should we be updating the samples also?  And for that matter, I don't matter whether the samples were updated for master branch.

Reply:
done